.github/workflows/viash-test.yml

```diff
@@ -27,7 +27,26 @@ jobs:
           pip install ruff
       - name: Run Ruff
         run: ruff check --output-format=github .
+      
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
 
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          packages: any::lintr, any::styler, any::roxygen2
+          needs: lint, styler
+
+      - name: Lint
+        run: lintr::lint_dir(path = ".")
+        shell: Rscript {0}
+        env:
+          LINTR_ERROR_ON_LINT: true
+
+      - name: Style
```

Comment:
Is this part necessary? If the previous step fails the user should run styler themselves, or do you intend the CI to create a PR for you?

Reply:
Ix, thanks!